core/src/main/java/org/axonframework/config/EventProcessorRegistry.java

```diff
@@ -0,0 +1,324 @@
+     * {@link org.axonframework.eventhandling.EventProcessor} of the given {@code name}. This
+     * overrides the default ErrorHandler configured through the {@link org.axonframework.config.Configurer}.
+     *
+     * @param name                The name of the event processor
+     * @param errorHandlerBuilder The {@link org.axonframework.eventhandling.ErrorHandler} to use for the
+     *                            {@link org.axonframework.eventhandling.EventProcessor} with the given {@code name}
+     * @return event processor registry for chaining purposes
+     */
+    EventProcessorRegistry configureErrorHandler(String name,
+                                                 Function<Configuration, ErrorHandler> errorHandlerBuilder);
+
+    /**
+     * Configures the builder function to create the Message Monitor for the {@link EventProcessor} of the given name.
+     * This overrides any Message Monitor configured through {@link Configurer}.
+     *
+     * @param name                  The name of the event processor
+     * @param messageMonitorBuilder The builder function to use
+     * @return event processor registry for chaining purposes
+     */
+    default EventProcessorRegistry configureMessageMonitor(String name,
+                                                           Function<Configuration, MessageMonitor<Message<?>>> messageMonitorBuilder) {
+        return configureMessageMonitor(name,
+                                       (configuration, componentType, componentName) -> messageMonitorBuilder
+                                               .apply(configuration));
+    }
+
+    /**
+     * Configures the factory to create the Message Monitor for the {@link EventProcessor} of the given name. This
+     * overrides any Message Monitor configured through {@link Configurer}.
+     *
+     * @param name                  The name of the event processor
+     * @param messageMonitorFactory The factory to use
+     * @return event processor registry for chaining purposes
+     */
+    EventProcessorRegistry configureMessageMonitor(String name, MessageMonitorFactory messageMonitorFactory);
+
+    /**
+     * Register a TrackingProcessor using default configuration for the given {@code name}. Unlike
+     * {@link #usingTrackingProcessors()}, this method will not default all processors to tracking, but instead only
+     * use tracking for event handler that have been assigned to the processor with given {@code name}.
+     * <p>
+     * Events will be read from the EventBus (or EventStore) registered with the main configuration
+     *
+     * @param name The name of the processor
+     * @return event processor registry for chaining purposes
+     */
+    default EventProcessorRegistry registerTrackingEventProcessor(String name) {
+        return registerTrackingEventProcessorUsingSource(name, Configuration::eventBus);
+    }
+
+    /**
+     * Registers a TrackingProcessor using the given {@code source} to read messages from.
```

Comment:
I believe it should state `TrackingEventProcessor` i.o. `TrackingProcessor`.

Reply:
I just now notice this happens more often in the interface. As that intentional?

Reply:
The comment resembles one which was present in `EventHandlingConfiguration`, but I agree, it should be `TrackingEventProcessor` instead.